src/TetrisGame/Cell.java

```diff
@@ -0,0 +1,105 @@
+package TetrisGame;
+
+import java.awt.*;
+import javax.swing.*;
+import static TetrisGame.GamePanel.convert;
+
+/**
+ * Created by yayixu on 9/11/17.
+ */
+public class Cell extends Component{
+    private int x, y, z;
+
+    public Cell(int x, int y, int z) {
+        this.x = x;
+        this.y = y;
+        this.z = z;
+    }
+
+    @Override
+    public void paint(Graphics g) {
```

Comment:
Like what we discussed earlier, you should have changed your Cell implementation to using JComponent, right? Also you need to override paintComponent instead of paint().